### DISCUSSION TITLE
how to start a wiki in a totally different directory

### DISCUSSION THREAD

**Discussion by wf5w** [gollum/gollum#1867] [Q&A]
I would like to start up gollum so that it starts in a totally different directory other than /home

I tried the --page-file-dir

--page-file-dir $HOME/dev/docs/markdown/linux/classes

It didn't work as I thought it would.

Any help on this?
Thanks.

**Comment by benjaminwil** [comment]
Hey. I was able to reproduce your issue:

My git repository root is at `$HOME/Wiki`.

So I started Gollum like this:

```
gollum $HOME/Wiki --page-file-dir $HOME/Wiki/Subdirectory
```

And nothing showed up. It seems that `--page-file-dir` only takes paths relative to the root. (I personally consider this not very straightforward. Maybe we should fix this.)

So to fix this I had to start Gollum like so:

```
gollum $HOME/Wiki --page-file-dir Subdirectory
```

Hope this helps.
